package.json

```diff
@@ -15,15 +15,16 @@
     "marked": "^0.3.12",
     "minimist": "^1.2.0",
     "sander": "^0.6.0",
+    "ts-node": "^8.3.0",
     "unicode-7.0.0": "^0.1.5",
     "xregexp": "^2.0.0"
   },
   "scripts": {
     "update-markdown-leafdoc": "src/cli.js -t templates/markdown src/cli.js src/leafdoc.js --verbose -o Leafdoc.md",
     "update-html-leafdoc": "src/cli.js -t templates/basic src/leafdoc.js src/cli.js --verbose -o Leafdoc.html",
-    "prepublish": "npm run update-markdown-leafdoc && npm run update-html-leafdoc",
-    "rollup": "rollup -c rollup.config.js",
-    "test": "rollup -c rollup.config.js; jasmine",
+    "TMP-DISABLED-prepublish": "npm run update-markdown-leafdoc && npm run update-html-leafdoc",
```

Comment:
I temporarily disabled this script, because it was preventing `npm install` from working. But now that `ts-node` is in place, it'll work. I'll update the PR...

Reply:
fixed.